### DISCUSSION TITLE
Git Server using Go-Git?

### DISCUSSION THREAD

**Discussion by PeterBocan** [go-git/go-git#1086] [General]
Did anyone manage to implement a Git server using Go-Git? I am guessing there is some functionality missing for the fully-fledged Git server...

**Comment by pjbgf** [comment]
There is an existing issue talking about this [here](https://github.com/go-git/go-git/issues/234).

One example for a Git server using `go-git` being: https://github.com/seankhliao/gitreposerver

**Comment by PeterBocan** [comment]
I haven't studied the Git protocol at all. Seems like it's a bit too simple to implement a git protocol. Is that all there is to it to the Git Protocol? Just uploading and downloading the chunks?
